object_storage_api/schemas/image.py

```diff
@@ -19,12 +19,20 @@ class ImagePostMetadataSchema(BaseModel):
     description: Optional[str] = Field(default=None, description="Description of the image")
 
 
-class ImageSchema(CreatedModifiedSchemaMixin, ImagePostMetadataSchema):
+class ImageMetadataSchema(CreatedModifiedSchemaMixin, ImagePostMetadataSchema):
     """
-    Schema model for an image get request response.
+    Schema model for an image's metadata.
     """
```

Comment:
This could be made into a single line docstring too. It won't let me write the change myself as it has a deleted line in it.